libretroshare/src/services/p3gxschannels.cc

```diff
@@ -1137,11 +1137,30 @@ bool p3GxsChannels::getChannelContent( const RsGxsGroupId& channelId,
 	msgIds[channelId] = contentIds;
 
 	if( !requestMsgInfo(token, opts, msgIds) || waitToken(token) != RsTokenService::COMPLETE )
-        return false;
+		return false;
 
 	return getPostData(token, posts, comments);
 }
 
+bool p3GxsChannels::getChannelComments(const RsGxsGroupId &channelId,
+                                       const std::set<RsGxsMessageId> &contentIds,
+                                       std::vector<RsGxsComment> &comments)
+{
+	std::vector<RsGxsGrpMsgIdPair> msgIds;
+	for (auto msg:contentIds)
```

Comment:
seems to me that for(auto& msg:comments) would avoid a copy, right?

Reply:
You're right.
I did not think about it.
Done